test/lib/ufe/testObject3d.py

```diff
@@ -320,7 +321,7 @@ def testVisibility(self):
         self.assertFalse(ufe.Object3d.hasObserver(visObs))
         self.assertEqual(ufe.Object3d.nbObservers(), 0)
 
-    @unittest.skipIf(os.getenv('UFE_PREVIEW_VERSION_NUM', '0000') < '2034', 'testUndoVisibleCmd is only available in UFE preview version 0.2.34 and greater')
+    @unittest.skipIf(ufeUtils.ufeFeatureSetVersion() >= 2, 'testUndoVisibleCmd only available in UFE v2 or greater.')
```

Comment:
Wrong way around --- should be skipUnless(), or < 2.

Reply:
Ah yes the dreaded skipIf vs skipUnless. Thanks for catching this.